x-pack/plugins/security_solution/server/lib/risk_engine/tasks/risk_scoring_task.test.ts

```diff
@@ -491,6 +507,20 @@ describe('Risk Scoring Task', () => {
           }
         });
 
+        it('does not schedules the transform to run now', async () => {
```

Comment:
It might be useful to add a unit test showing that we do schedule the transform in the case where the task was cancelled 👍

Reply:
Good call!